src/settings/BatchGroupConfigurationSettings/BatchGroupCongiurationForm.js

```diff
@@ -45,11 +188,28 @@ BatchGroupConfigurationForm.propTypes = {
   batchGroups: PropTypes.arrayOf(PropTypes.object),
   handleSubmit: PropTypes.func.isRequired,
   pristine: PropTypes.bool,
-  setSelectedBatchGroupId: PropTypes.func.isRequired,
+  selectBatchGroup: PropTypes.func.isRequired,
   submitting: PropTypes.bool,
+  form: PropTypes.object,
+  selectedBatchGroupId: PropTypes.string,
 };
 
 export default stripesFinalForm({
-  navigationCheck: true,
   subscription: { values: true },
```

Comment:
FYI, you have subscription to values, but you don't use it

Reply:
it's still active

Reply:
It needs for `On change` listener